internal/cmd/launcher.go

```diff
@@ -123,7 +119,10 @@ func (b *customBinary) run(gs *state.GlobalState) error {
 	// the subprocess detects the type of terminal
 	cmd.Stderr = os.Stderr
 	cmd.Stdout = os.Stdout
-	cmd.Stdin = os.Stdin
+
+	// if we are here, it is possible that the analyze function read the stdin to analyse it
```

Comment:
nit (based on my current understanding, I might have gotten it wrong 🙇🏻 )

```suggestion
  // If stdin was used by the analyze function, the content has been preserved
  // in `gs.Stdin` and should be passed to the command
```